static/js/app.js

```diff
@@ -686,16 +686,59 @@ if ( document.getElementById("survey-dismiss") ) {
 }
 
 
-//Landing page use case accordion 
+//Landing page use case accordion
+const useCaseSection = document.getElementById("use-cases");
 const useCaseTitle = document.querySelectorAll(".use-case-title");
 
 function toggleClass(elem) {
    useCaseTitle.forEach( item => {
       item.classList.remove("is-active");
   });
+  useCaseSection.scrollIntoView();
   elem.target.classList.add("is-active");
+  window.location.hash = "#use-cases" + "/" + elem.target.dataset.useCase;
 }
 
+function hashChangeAccordion(){
+  const urlHash = location.hash;
+  if (urlHash.includes("#use-cases")){
+    useCaseSection.scrollIntoView();
+    
+    if (urlHash === "#use-cases/shopping"){
+      useCaseTitle.forEach( item => {
+        item.classList.remove("is-active");
+      });
+      document.querySelector(".use-case-shopping").classList.add("is-active");
```

Comment:
nit: Since we're repeating this so much, I recommend moving it to its own function — that way we can change it once and update each if. 

You could even pass in the class name below to clean that up too — 

```suggestion
      useCaseTitle.resetAccordianAndSetActiveSection(".use-case-shopping")
```